internal/storage/resource_storage_volume.go

```diff
@@ -167,6 +194,17 @@ func (r StorageVolumeResource) Create(ctx context.Context, req resource.CreateRe
 		return
 	}
 
+	if !plan.SourceVolume.IsNull() {
+		r.copyStoragePoolVolume(ctx, resp, &plan)
+		return
+	} else {
+		r.createStoragePoolVolume(ctx, resp, &plan)
+		return
+	}
+
```

Comment:
This newline can be removed

Reply:
> This newline can be removed

Thank you for pointing that out; the issue has been resolved.